jpsreport/methods/Method_A.cpp

```diff
@@ -134,21 +129,20 @@ void Method_A::GetAccumFlowVelocity(
     const vector<double> & VInFrame)
 {
     for(unsigned int i = 0; i < ids.size(); i++) {
-        int id          = ids[i];
         bool IspassLine = false;
-        if(frame > _firstFrame[id - _minId] && !_passLine[id - _minId]) {
+        if(frame > _firstFrame[i] && !_passLine[i]) {
```

Comment:
@schroedtert Based on our discussion: Does this improve memory management?

Reply:
Not all, for memory improvement the whole structure needs to changed. But that is not that important.

Reply:
ok